doc/examples/test_examples.py

```diff
@@ -52,3 +52,11 @@ def test_example(example):
     else:
         assert output == baseline
     os.chdir(startdir)
+
+
+def test_cmake():
+    startdir = os.getcwd()
+    os.chdir(datadir)
+    subprocess.check_call('./example_cmake.sh')
+    # TODO: Verify the output
```

Comment:
See my other comment. I can see 'scrubbers' mentioned in this file, but I don't know how much is really compared. `cmake` and `make` are not silent and will produce system-dependent output. However, it would be good to check that the final code coverage number is correct.

Reply:
I've implemented the checking of the output: stuff we are not interested in can just be redirected to STDERR and will not be captured. The scrubbing system is a bunch of regexes to normalize the output, but it's not necessary here. This comes with two caveats:

1. The example tests are skipped on Windows (mainly because I have no means to debug MinGW problems)
2. The cmake test now uses relative paths instead of absolute paths. The problem with absolute paths is that they are different on each system, which is a problem for testing.